### PR TITLE
Fix CI warnings

### PR DESCRIPTION
**Pull request by vincentisambart**
Looking at existing PRs, the CI warnings were distracting so I fixed them.

### CODE REVIEW

File: src/backend/mac/menu.rs [hunk 1]
```diff
@@ -159,7 +159,7 @@ impl HotKey {
             KbKey::F19 => "\u{F716}",
             KbKey::F20 => "\u{F717}",
             _ => {
-                eprintln!("no key equivalent for {:?}", self);
+                eprintln!("no key equivalent for {self:?}");
```
Comment (vincentisambart) [inline src/backend/mac/menu.rs:162]: That requires Rust 1.58 but I couldn't find any mention of the MSRV, so I guess this should be fine?
If that's not the case, the `uninlined_format_args` clippy lint should be disabled instead.

File: src/backend/windows/window.rs [hunk 2]
```diff
@@ -187,8 +187,8 @@ unsafe impl HasRawWindowHandle for WindowHandle {
 
 unsafe impl HasRawDisplayHandle for WindowHandle {
     /// See:
-    ///  *  https://github.com/rust-windowing/raw-window-handle/issues/92
-    ///  * https://github.com/rust-windowing/winit/blob/92fdf5ba85f920262a61cee4590f4a11ad5738d1/src/platform_impl/windows/window.rs#L285
+    ///  * <https://github.com/rust-windowing/raw-window-handle/issues/92>
+    ///  * <https://github.com/rust-windowing/winit/blob/92fdf5ba85f920262a61cee4590f4a11ad5738d1/src/platform_impl/windows/window.rs#L285>
```
Comment (vincentisambart) [inline src/backend/windows/window.rs:191]: To fix rustdoc's `bare_urls` warning.
Rustdoc does not automatically turn bare URLs into links, they need to be wrapped with angle brackets.

File: src/screen.rs [hunk 3]
```diff
@@ -21,7 +21,7 @@ use std::fmt::Display;
 
 /// Monitor struct containing data about a monitor on the system
 ///
-/// Use Screen::get_monitors() to return a Vec<Monitor> of all the monitors on the system
+/// Use `Screen::get_monitors()` to return a `Vec<Monitor>` of all the monitors on the system
```
Comment (vincentisambart) [inline src/screen.rs:24]: To fix rustdoc's `invalid_html_tags` warning.
`<Vector>` is seen as an (invalid) HTML tags if not inside backquotes.

File: src/backend/mac/menu.rs (hunk 1 above)
Comment (xStrom) [reply src/backend/mac/menu.rs:162]: It's not explicitly defined, but right now during heavy development we're basically fine with treating whatever is currently the stable version as our MSRV.

**Comment by xStrom** [comment]
Thanks for doing this!

I see that `examples/shello.rs` was not treated. Is this on purpose (*which is fine, we can do it later*) or would you like to also fix that?

**Comment by vincentisambart** [comment]
> I see that `examples/shello.rs` was not treated. Is this on purpose (_which is fine, we can do it later_) or would you like to also fix that?

Sorry, but I could not find any other warning when building `example/shello.rs` 🤔

**Comment by xStrom** [comment]
It's visible at the bottom of the [files changed](https://github.com/linebender/glazier/pull/60/files) page.

**Comment by vincentisambart** [comment]
> It's visible at the bottom of the [files changed](https://github.com/linebender/glazier/pull/60/files) page.

Oops, indeed 😅. I hadn't seen it on the other PRs, and hadn't realized you let the CI run on this one.
Just fixed it.

**Review by xStrom — APPROVED** [review]
Thanks again for getting this done!

**Pull request merged**